dist/msrcrypto.js

```diff
@@ -9107,7 +9107,7 @@ function buildParameterCollection(operationName, parameterSet) {
         var actualParam = parameterSet[i];
 
         // Verify the required parameters are present.
-        if (!actualParam) {
+        if (actualParam===undefined) {

```

Comment:
This will miss `actualParam === null`. To check for `null` or `undefined`, use `actualParam == null`.

Reply:
done